core/build.gradle

```
@@ -53,7 +63,47 @@ animalsniffer {
 
 import net.ltgt.gradle.errorprone.CheckSeverity
 
+def replaceBytes(byte[] haystack, byte[] needle, byte[] replacement) {
+    int i = Bytes.indexOf(haystack, needle);
+    assert i != -1;
+    byte[] result = new byte[haystack.length - needle.length + replacement.length];
+    System.arraycopy(haystack, 0, result, 0, i);
+    System.arraycopy(replacement, 0, result, i, replacement.length);
+    System.arraycopy(haystack, i + needle.length, result, i + replacement.length, haystack.length - i - needle.length);
+    return result;
+}
+
+def bigEndian(int value) {
```

Maybe something like
```suggestion
def bigEndianBytesShort(int value) {
```

To indicate this only works with 2-byte ints?

---

I went ahead and used the name `bigEndianShortBytes()`